backend/src/services/v2/review.ts

```diff
@@ -166,39 +166,47 @@ export async function sendReviewResponseNotification(review: ReviewDoc, user: Us
       }
 
       const access = await getAccessRequestById(user, review.accessRequestId)
-      notifyReviewResponseForAccess(review, access)
+      try {
+        notifyReviewResponseForAccess(review, access)
```

Comment:
This is an async function that's not being 'awaited', therefore the `try {} catch{}` won't do anything here.  Either await this, or turn the try / catch into a `.catch()` statement.